From khali@linux-fr.org Sat Nov 26 19:24:47 2005
Date: Sat, 26 Nov 2005 20:52:34 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Laurent Riffard <laurent.riffard@free.fr>, Petr Vandrovec <vandrove@vc.cvut.cz>
Subject: [PATCH 18/25] i2c: Drop i2c_driver.{owner,name}, 9 of 11
Message-Id: <20051126205234.70e635ef.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-drop-driver-owner-and-name-09-video.patch

From: Laurent Riffard <laurent.riffard@free.fr>

We should use the i2c_driver.driver's .name and .owner fields
instead of the i2c_driver's ones.

This patch updates the matroxfb driver.

Signed-off-by: Laurent Riffard <laurent.riffard@free.fr>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Acked-by: Petr Vandrovec <petr@vandrovec.name>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/video/matrox/matroxfb_maven.c |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

--- gregkh-2.6.orig/drivers/video/matrox/matroxfb_maven.c
+++ gregkh-2.6/drivers/video/matrox/matroxfb_maven.c
@@ -1293,8 +1293,10 @@ static int maven_command(struct i2c_clie
 }
 
 static struct i2c_driver maven_driver={
-	.owner		= THIS_MODULE,
-	.name		= "maven",
+	.driver = {
+		.owner	= THIS_MODULE,
+		.name	= "maven",
+	},
 	.id		= I2C_DRIVERID_MGATVO,
 	.attach_adapter	= maven_attach_adapter,
 	.detach_client	= maven_detach_client,
